src/wp-admin/includes/class-wp-comments-list-table.php

```diff
@@ -864,6 +864,12 @@ protected function handle_row_actions( $item, $column_name, $primary ) {
 
 		$output .= '<button type="button" class="toggle-row"><span class="screen-reader-text">' . __( 'Show more details' ) . '</span></button>';
 
+		if ( isset( $actions['quickedit'] ) || isset( $actions['quickedit hide-if-no-js'] ) ) {
+			ob_start();
+				wp_comment_reply( '-1', true, 'detail' );
```

Comment:
```suggestion
			wp_comment_reply( '-1', true, 'detail' );
```
Looks like there's an extra tab here.